cmd/expired-authz-purger/main.go

```diff
@@ -60,44 +88,38 @@ func (p *expiredAuthzPurger) purge(table string, purgeBefore time.Time, parallel
 		query = "SELECT id FROM authz WHERE id >= :id AND expires <= :expires ORDER BY id LIMIT :limit"
 	}
 
-	done := make(chan int)
 	work := make(chan string)
 	go func() {
 		// id starts as "", which is smaller than all other ids.
 		var id string
 		var count int
-		for count < max {
-			var idBatch []string
-			_, err := p.db.Select(
-				&idBatch,
-				query,
-				map[string]interface{}{
-					"id":      id,
-					"expires": purgeBefore,
-					"limit":   p.batchSize,
-				},
-			)
-			if err != nil && err != sql.ErrNoRows {
-				p.log.AuditErrf("Getting a batch: %s", err)
-				time.Sleep(10)
```

Comment:
Note: I've changed this from `time.Sleep(10)` which is confusing as it uses no units, and sleeps for 10ns which I don't think was originally intended to `time.Sleep(time.Millisecond * 500)` which seems more realistic.